lib/global_config.js

```diff
@@ -256,4 +260,19 @@ exports.getKeepAlive = function () {
   return keepAlive;
 };
 
+let credentialManager = createJsonFileCredentialManager();
+
+exports.setCustomCredentialManager = function (customCredentialManager) {
+  Errors.checkArgumentValid(Util.checkValidCustomCredentialManager(customCredentialManager),
+  ErrorCodes.ERR_GLOBAL_CONFIGURE_INVALID_CUSTOM_CREDENTIAL_MANAGER);
+  
+  credentialManager =  customCredentialManager;
+};
+
+exports.getCredentialManager = function () {  
+  return credentialManager;  
+};  
 
+function createJsonFileCredentialManager() {
```

Comment:
what do you think about `createDefaultCredentialManager()`?

Reply:
More reasonable.

Reply:
Fixed